contrib/charts/dragonfly/ci/tls-values.golden.yaml

```diff
@@ -123,7 +123,7 @@ spec:
             - mountPath: /etc/dragonfly/tls
               name: tls
           env:
-            - name: DFLY_PASSWORD
+            - name: DFLY_requirepass
```

Comment:
Can you add another test config (amnd golden file) for `.Values.image.tag` older than v1.14 so that we can verify the same?

Mostly looks good otherwise! :)

Reply:
Thanks for the review. I just pushed a commit that I believe addresses this request.